umbraco-cloud/set-up/project-settings/manage-hostnames/rewrites-on-cloud.md

```diff
@@ -145,3 +145,6 @@ The following redirect is a way to amend the issue where the `.azurewebsites.net
   <action type="Redirect" url="https://www.hostname.com/{R:0}" appendQueryString="true" redirectType="Permanent" />
 </rule>
 ```
+{% hint style="warning" %}
+Please keep in mind that this can only be done for projects with only one custom hostname. Setting up multiple rules with different domains will result in an internal server error. 
```

Comment:
A suggestion to leave out the part about the error. Aren't we implicitly saying that there will be an error, when we mention that it will only work when having a single custom hostname configured.

```suggestion
The redirect for `.azurewebsites.net` can be used on projects where only one custom hostname is configured.
```